ddl/reorg.go

```diff
@@ -252,10 +253,6 @@ func (w *worker) runReorgJob(rh *reorgHandler, reorgInfo *reorgInfo, tblInfo *mo
 		}
 
 		updateBackfillProgress(w, reorgInfo, tblInfo, 0)
-		if err1 := rh.RemoveDDLReorgHandle(job, reorgInfo.elements); err1 != nil {
-			logutil.BgLogger().Warn("[ddl] run reorg job done, removeDDLReorgHandle failed", zap.Error(err1))
-			return errors.Trace(err1)
-		}
```

Comment:
This is where the conflict was created. The tidb_ddl_reorg table had already been changed in the reorg work function `f()` given as an argument. While this change is in the scope of the `w.sess` session which was started before calling this function and committed after returning. It has been replaced by `CleanupDDLReorgHandles` in `HandleDDLJobDone` and `HandleDDLJobTable`.